### PR TITLE
ALFREDAPI-505

### PR DESCRIPTION
**Pull request by WimCrols**
Fixes https://xenitsupport.jira.com/browse/ALFREDAPI-505

- [x] Is [CHANGELOG.md](https://github.com/xenit-eu/alfred-api/blob/master/CHANGELOG.md) extended?
- [ ] Does this PR avoid breaking the API? 
    Breaking changes include adding, changing or removing endpoints and/or JSON objects used in requests and responses.
- [x] Does the PR comply to REST HTTP result codes policy outlined in the [user guide](https://docs.xenit.eu/alfred-api/stable-user/rest-api/index.html#rest-http-result-codes)?
- [x] Is error handling done through a method annotated with `@ExceptionHandler` in the webscript classes?
- [x] Does the PR follow our [coding styleguide and other active procedures](https://xenitsupport.jira.com/wiki/spaces/XEN/pages/624558081/XeniT+Enhancement+Proposals+XEP)?
- [x] Is usage of `this.` prefix avoided?

See [README.md](./README.md) for full explanation.

[ALFREDAPI-505]: https://xenitsupport.jira.com/browse/ALFREDAPI-505?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -45,23 +45,26 @@ node {
...
+//         }
+//         stage("Build 70") {
+//             BuildVersionX("70")
+//         }
+//         stage("Build 71") {
+//             BuildVersionX("71")
+//         }
+//         stage("Build 72") {
+//             BuildVersionX("72")
+//         }
```
Comment (RVanhuysseXenit) [inline Jenkinsfile]: Dont forget to uncomment

File: apix-docker/73/build.gradle [hunk 2]
```diff
@@ -0,0 +1,8 @@
+dependencies {
+    baseAlfrescoWar platform("org.alfresco:acs-packaging:7.3.0.1")
+    baseAlfrescoWar 'org.alfresco:content-services@war'
+}
+
+dockerAlfresco {
+    baseImage = 'docker.xenit.eu/private/alfresco-enterprise/alfresco-repository-enterprise:7.3.0.1'
```
Comment (RVanhuysseXenit) [inline apix-docker/73/build.gradle:7]: can be updated to use `private.docker.xenit.eu/alfresco-enterprise/alfresco-repository-enterprise`.
Also, I thought the tag with the label component was not available?

File: apix-docker/73/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      - alfresco:/opt/alfresco/alf_data
+    restart: unless-stopped
+    environment:
+      - SOLR_HOST=solr
+      - TERM=xterm
+      - GLOBAL_messaging.broker.url=failover:(nio://activemq:61616)?timeout=3000&jms.useCompression=true
+      - GLOBAL_localTransform.core-aio.url=http://transform-core-aio:8090/
+
+  solr:
+    image: hub.xenit.eu/alfresco-enterprise/alfresco-solr6:2.0.1
```
Comment (RVanhuysseXenit) [inline apix-docker/73/docker-compose.yml:18]: Could use solr6-2.0.3 ?

**Review by RVanhuysseXenit — APPROVED** [review]

File: apix-docker/73/build.gradle (hunk 2 above)
Comment (WimCrols) [reply apix-docker/73/build.gradle:7]: Yes, I think I'll migrate to GHA, but seems cleaner in a separate ticket.

**Review by todorinskiz — APPROVED** [review]

**Pull request merged**